### DISCUSSION TITLE
Simple way to plot pseudotime on heatmap columns

### DISCUSSION THREAD

**Discussion by mitchTwoTimes** [GreenleafLab/ArchR#1579] [Feature Requests]
Hi,

Is there a way to annotate the columns of `plotTrajectoryHeatmap` heatmaps with pseudotime values? I'm sure this is a matter of digging through `ComplexHeatmap` vignettes, but an ArchR command could make it easier to visualize pseudotemporal changes in a feature of interest. 

Thanks!

**Comment by annrosebright** [comment]
I have the same question. Is there any way to extract column information to plot the matrix using a heatmap?

**Comment by rcorces** [comment]
Unless I'm mis-reading the question, the heatmap is features (rows) by pseudotime bins (columns). And the number of pseudotime bins depends on `groupEvery` from getTrajectory. And the bins are ordered. So you already know the pseudotime values of the columns and this information would be redundant

**Reply by annrosebright** [reply]
Okay, thanks for your reply. I understand now. 
I was looking/hoping to find a way to associate the bins with the clusters so that I can annotate the columns as clusters. This helps in interpreting the results.

**Reply by rcorces** [reply]
you mean like this? (available on `dev` branch)
https://github.com/GreenleafLab/ArchR/discussions/580
